tests/installation/releasenotes.pm

```diff
@@ -23,7 +23,7 @@ sub run() {
     }
 
     # workaround for bsc#1014178
-    if (check_screen('zfcp-popup'), 0) {
+    if (check_screen('zfcp-popup'), 5) {
```

Comment:
The idea behind `check_screen(…), 0` was to assert that the screen is only checked as is at the time of call, no waiting. If you need to wait longer then maybe 5 seconds is also not enough. I recommend to either look for all alternatives with `assert_screen` and at least default timeout or the "lazy" approach:

```perl
wait_still_screen(3);
if (check_screen('zfcp-popup'), 0) {
…
}
```